src/client/src/components/pages/participants/participantDetails/staticData.js

```diff
@@ -41,7 +42,14 @@ export const fields = [
       value: "empty",
       color: "#FF4800",
       staticField: true
-    },  {
+    },  
+    {
+      tag: "Button",
+      value: "Prison Details",
+      display: "none",
+      color: "#FF4800",
+      staticField: true
+    },{
```

Comment:
``` },{``` looks wrong and inconsistent with the other objects in this array. This suggests to me that ESLint is not working, as the formatting stuff should be fixed automatically

Reply:
i do it